examples/unity/ios/README.md

```diff
@@ -0,0 +1,79 @@
+# Insertable Streams iOS Unity sample
+This sample application shows how to integrate insertable streams (video) with unity.
+Integrating this sample with **Vonage Video SDK** will allow you to publish a unity scene as your video. 
```

Comment:
This is not really correct. We are not using/integrating this with the video client SDKs.